platform/auth-monitor/auth-monitor.go

```diff
@@ -11,16 +11,16 @@ import (
 	"crypto/tls"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/sipb/homeworld/platform/keysystem/api"
+	"github.com/sipb/homeworld/platform/keysystem/api/reqtarget"
+	"github.com/sipb/homeworld/platform/keysystem/api/server"
+	"github.com/sipb/homeworld/platform/util/osutil"
```

Comment:
all go files: should organize commits into sections for clarity

Reply:
Will resolve this in a follow-up PR, for the sake of getting the core changes merged ASAP.